csss-site/src/events/tech_fair/templates/tech_fair/2022/tech_fair_main.html

```diff
@@ -0,0 +1,177 @@
+{% load staticfiles %}
+
+<!DOCTYPE html>
+<html lang="en">
+
+
+	<meta charset="UTF-8">
+	<link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap@4.3.1/dist/css/bootstrap.min.css" integrity="sha384-ggOyR0iXCbMQv3Xipma34MD+dH/1fQ784/j6cY/iJTQUOhcWr7x9JvoRxT2MZw1T" crossorigin="anonymous">
+	<link rel="stylesheet" type="text/css" href="{% static 'tech_fair_css.css' %}"/>
```

Comment:
please match existing convention w/ regards to naming the `.css` files

Reply:
Fixed. Resolved.